kola-denylist.yaml

```diff
@@ -23,5 +23,7 @@
   osversion:
     - c9s
 
-- pattern: iso-offline-install-iscsi.bios
-  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1638
+# This test is failing only in prow, so it's skipped by prow
```

Comment:
Minor: a bit awkward to have a commented-out denylist entry. I think it'd be clearer to instead add the link to https://github.com/openshift/os/issues/1492 as a comment just above where we `--denylist-test` it in `ci/prow-entrypoint.sh` since that's where the denying actually happens?

Reply:
This was meant to be a reminder that we skip this test is prow. `kola-denylist.yaml`  get more eyes than ci-entrypoint I suppose. 
I'm fine changing it if you have strong opinions :)

Reply:
Not a strong opinion either. :)